.devcontainer/devcontainer.json

```diff
@@ -32,6 +32,9 @@
         "eamodio.gitlens",
         "esbenp.prettier-vscode",
         "mhutchie.git-graph",
+        "ms-python.python",
+        "ms-python.black-formatter",
+        "ms-python.flake8",
```

Comment:
Can we add 

```
"batisteo.vscode-django",
"monosans.djlint",
"qwtel.sqlite-viewer",
```

Reply:
Yep, added in f20297eeb8aab92500d0209ce55ef0e1e5712e92